hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/staking/StakingRewardsHandlerImpl.java

```diff
@@ -244,19 +244,21 @@ private void adjustNodeStakes(
             final Instant consensusNow) {
         if (scenario.withdrawsFromNode()) {
             final var currentStakedNodeId = originalAccount.stakedNodeId();
-
-            stakeInfoHelper.withdrawStake(currentStakedNodeId, originalAccount, stakingInfoStore);
-            if (containStakeMetaChanges) {
-                // Pending rewards are calculated midnight each day for every account.
-                // If this account has changed to a different stakeId or choose to decline reward
-                // in mid of the day, it will not receive rewards for that day.
-                // So, it will be leaving some rewards from its current node unclaimed.
-                // We need to record that, so we don't include them in the pendingRewards
-                // calculation later
-                final var effectiveStakeRewardStart = rewardableStakeStartFor(
-                        stakingRewardStore.isStakingRewardsActivated(), originalAccount, consensusNow);
-                stakeInfoHelper.increaseUnclaimedStakeRewards(
-                        currentStakedNodeId, effectiveStakeRewardStart, stakingInfoStore);
+            // -1 is a special value to remove the account's staked node ID.
+            if (currentStakedNodeId != -1) {
```

Comment:
same as above , will be nice to use static variable with clear name for -1

Reply:
updated